core/src/main/java/jenkins/formelementpath/FormElementPathPageDecorator.java

```diff
@@ -0,0 +1,20 @@
+package jenkins.formelementpath;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import hudson.Extension;
+import hudson.Main;
+import hudson.model.PageDecorator;
+import jenkins.util.SystemProperties;
+
+@Extension
+public class FormElementPathPageDecorator extends PageDecorator {
+
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
+    private static /*almost final */ boolean ENABLED = Main.isUnitTest ||
+            SystemProperties.getBoolean(FormElementPathPageDecorator.class.getName() + ".enabled");
```

Comment:
possible future enhancement would be to also set this for `RealJenkinsRule`

Reply:
That should set `isUnitTest` IIRC.